init.sh

```diff
@@ -105,8 +112,8 @@ EOT
 fi
 
 # Check for input parameters
-if [[ (-z $CLUSTER_TYPE) || (-z $DEPLOY_REGION) || (-z $ACCOUNT_ID) || (-z $CLUSTER_SIZE) \
-   || (-z $RANDOM_STR) || (-z $BASE_DOMAIN) || (-z $SSH_KEY_NAME) || (-z $DEPLOY_WAIT_HANDLE) ]]; then
+if [[ (-z $CLUSTER_TYPE) || (-z $DEPLOY_REGION) || (-z $ACCOUNT_ID) \
```

Comment:
Since this if condition is common, check for all offerings Removing `CLUSTER_SIZE `and `BASE_DOMAIN` will still pass PRE_VALIDATION in other scenarios where they are mandatory.

Reply:
Not exactly, as that can be checked in the respective `CFT`s . Previously, since we had only 1 CFT _(ok 2 if we consider prod/dev environments)_ it was more relevant. However, now that we have multiple `CFT`s we can check the condition in those `CFT`s itself

Reply:
Since the mandatory checks can be handled in the CFT, this change looks good.